Dockerfile

```diff
@@ -43,12 +43,12 @@ RUN mkdir /ucentral-data
 RUN apk add --update --no-cache librdkafka mariadb-connector-c libpq unixodbc
 
 COPY --from=builder /ucentralgw/cmake-build/ucentralgw /ucentral/ucentralgw
-COPY --from=builder /cppkafka/cmake-build/src/lib/* /lib/
+COPY --from=builder /cppkafka/cmake-build/src/lib64/* /lib/
 COPY --from=builder /poco/cmake-build/lib/* /lib/
 
 EXPOSE 15002
-EXPOSE 15015
-EXPOSE 16001
+EXPOSE 16002
```

Comment:
The internal endpoint port `17002` is missing. But I also did the changes to the Dockerfile in my PR, so maybe its best if only one of us adapts the changes so that we don't overwrite each other.

Reply:
Sure, let's use Dockerfile from your branch. I'll take only care about Helm chart